.circleci/config.yml

```diff
@@ -205,14 +206,12 @@ jobs:
         environment:
           POSTGRES_USER: root
           POSTGRES_DB: circle_test
-
-
     steps:
       - checkout
       # removing as we deal with caching problem
       # - restore_cache:
       # # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
-      #     key: deps-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ checksum "Pipfile.lock" }}-{{ checksum "package.json" }}
+      #     key: deps-v4-{{ .Branch }}-{{ checksum "Pipfile.lock" }}-{{ checksum "package.json" }}
```

Comment:
Do we need this commented out code?

Reply:
I haven't looked at what this does, but I'm okay with leaving it alone if you are.

Reply:
Works for me!